test/controllers/user.test.js

```diff
@@ -9,15 +9,15 @@ const { factoryAllModels } = require('../factory/factory_by_models');
 factoryAllModels();
 
 const request = supertest(app);
-const userAttributes = (firstName, lastName, email, password) => ({
-  firstName,
-  lastName,
+const userAttributes = (first_name, last_name, email, password) => ({
+  first_name,
+  last_name,
```

Comment:
Function params should never ever be in snake_case.
I know it's annoying, but do:
```
const userAttributes = (firstName, lastName, email, password) => ({
  first_name:   firstName,
  last_name:  lastName,
  ...
```